x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/malware.tsx

```diff
@@ -181,26 +201,55 @@ export const MalwareProtections = React.memo(() => {
           })}
         </ProtectionRadioGroup>
         <EuiSpacer size="s" />
-        <EuiTitle size="xxxs">
-          <h6>
+        <EuiTitle size="xxs">
+          <h5>
             <FormattedMessage
               id="xpack.securitySolution.endpoint.policyDetailsConfig.userNotification"
               defaultMessage="User Notification"
             />
-          </h6>
+          </h5>
         </EuiTitle>
         <EuiCheckbox
           id="xpack.securitySolution.endpoint.policyDetail.malware.userNotification"
           onChange={handleUserNotificationCheckbox}
           checked={userNotificationSelected}
           label={i18n.translate(
-            'xpack.securitySolution.endpoint.policyDetail.malware.userNotification',
+            'xpack.securitySolution.endpoint.policyDetails.malware.userNotification',
             { defaultMessage: 'User Notification' }
           )}
         />
+        {userNotificationSelected && (
+          <>
+            <EuiSpacer size="s" />
+            <EuiText size="xs">
+              <h4>
+                <FormattedMessage
+                  id="xpack.securitySolution.endpoint.policyDetailsConfig.customizeUserNotification"
+                  defaultMessage="Customize notification message"
+                />
+              </h4>
+            </EuiText>
+            <EuiSpacer size="xs" />
+            <EuiTextArea
+              placeholder={i18n.translate(
+                'xpack.security_solution.endpoint.policyDetails.malware.userNotification.default',
```

Comment:
we actually do have a default message that we should send down.

From the ticket: `Elastic Security {action} {filename}`

Then, we should figure out some added explanation in the UI to describe the "{}" options.  We can do that in a subsequent PR.

In the meantime, though, we should send the default message as shown above.

Reply:
oh gotcha, yeah i do remember reading that somewhere, i just forgot once i was looking at the mocks

Reply:
That's going to be interesting - defining that default message via `i18n` being that it also uses `{}` for replaceable tokens 🙈 (wondering if `i18n` `defaultMessage` supports an escape sequence - let us know what you find @parkiino )

Re: Defaults

@kevinlog we should revisit how we want to handle "defaults".

❓ **Should the default value be left to the endpoint to use?**
When set the message in the Policy in kibana, even if that message matches what the endpoint's default (as of today), is that really the default or a custom message?  Should we (kibana) instead not set anything on the policy in kibana, and allow the endpoint to use whatever the default is that they have. This would be beneficial in case the Endpoint changes the default (ex. in addition to `{action}` and `{filename}`, they add a `{hash}`), but that value would never be seen because all of the kibana created policies have a specific value set for the message (which used to be the default, but not anymore).
We would also need to know what would trigger the Endpoint to display their default (ex. property is in the JSON, but not set (empty string); property is not in the JSON at all; Property is present but value is `undefined`)

_ps. The "Input message editor" component/widget sounds like a fun one to build_ 😃

Reply:
@paul-tavares if we initialize the default value to `undefined` or an empty string then the Endpoint will just use its own internal default value. (@ferullo correct me if I'm wrong, but I remember you said that `""` could be an OK default).

However, I wonder if there's value in showing the user what the default value is in the UI even if they've never edited it.  This would let them know what the Endpoint is going to use as the popup and sort of educates them about `{action}`, etc.

I could go either way or merge this and then figure out what that default is.

Let me know your thoughts @parkiino

Reply:
We could just leave the value as an empty string and have either the placeholder or maybe a tooltip (or some other messaging) to tell the user what the default message looks like. This would be in place of having the UI pre-populate the value with `Elastic Security { action } { filename } `, since the endpoint will still use the default notification if we send an empty string as the value.

Reply:
Correct. `""` is a valid default. Endpoint will treat the empty string as no-value-given and use a hard coded default message.